test/e2e/kb/version_upgrade_test.go

```diff
@@ -20,15 +20,21 @@ import (
 )
 
 func TestVersionUpgrade(t *testing.T) {
```

Comment:
For consistency with ES, it would be nice to test:
* from min 6.x to current 7.x
* from current 7.x to latest 7.x